src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -185,6 +185,8 @@ public void saveCountyCourtJudgment(String externalId, CountyCourtJudgment count
     }
 
     public void saveDefendantResponse(long claimId, String defendantId, String defendantEmail, Response response) {
+        // When this is saved in CCD ensure a Forbidden response is returned to the client if they
```

Comment:
What do you think about adding a story for this/updating an existing story to do this/at least adding a `TODO` tag?

Reply:
its right on a line that has to change when we save the defendant response